benchmarks-jmh/build.gradle

```diff
@@ -2,12 +2,15 @@ apply plugin:'application'
 
 mainClassName = 'org.openjdk.jmh.Main'
 
+sourceCompatibility = 1.8
```

Comment:
@morj Chronicle Map is Java 8+. Probably language level might be lower but it depends on JDK 1.8.